ratis-logservice/src/main/java/org/apache/ratis/logservice/server/MetaStateMachine.java

```diff
@@ -291,7 +290,8 @@ public TransactionContext preAppendTransaction(TransactionContext trx) throws IO
                 MetaServiceProtoUtil.toDeleteLogReplyProto().toByteString()));
     }
 
-//    private CompletableFuture<Message> processCloseLog(MetaServiceProtos.MetaServiceRequestProto logServiceRequestProto) {
```

Comment:
Please just delete commented out code instead of fixing it.

Reply:
Sure, I will post a revision shortly.